changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## ??? January 2023
+
+* Extended [Add order](../operations/orders.md#add-order) with `AccountId` property to support [Company](../operations/companies.md#company) accounts. `CustomerId` property was deprecated.
```

Comment:
I thinking it change it little bit to be more similar style to others. Any opinions? @jonas-paul @MikeAdamsMews 

```suggestion
* Deprecated `CustomerId` in request parameters of [Add order](../operations/orders.md#add-order) and replaced it with `AccountId` to support [Company](../operations/companies.md#company).
```